### DISCUSSION TITLE
Suggestions for Vector DBs to be added

### DISCUSSION THREAD

**Discussion by dhruv-anand-aintech** [superlinked/VectorHub#197] [VDB Comparison]
This is where the community can request for a specific Vector DB to be added to the table.

**Comment by dhruv-anand-aintech** [comment]
Upstash https://upstash.com/blog/introducing-vector-database

**Comment by dhruv-anand-aintech** [comment]
Infinity https://github.com/infiniflow/infinity

**Comment by dhruv-anand-aintech** [comment]
Bigquery https://www.linkedin.com/posts/mehd-io_introduction-to-vector-search-bigquery-activity-7162464747433492480-1UZI?utm_source=share&utm_medium=member_android

**Comment by dhruv-anand-aintech** [comment]
HollowDB https://github.com/firstbatchxyz/hollowdb-vector

**Comment by dhruv-anand-aintech** [comment]
Databricks vector search

**Comment by dhruv-anand-aintech** [comment]
pgvecto.rs

**Comment by dhruv-anand-aintech** [comment]
vlite
https://twitter.com/sdand/status/1776273741510832316?t=BCNzqN1SeAn920Q7U77nIg&s=19

**Comment by dhruv-anand-aintech** [comment]
Upstash https://upstash.com/docs/vector/overall/getstarted

**Comment by dhruv-anand-aintech** [comment]
Tembo https://tembo.io/

**Comment by MattiaIppoliti** [comment]
AlloyDB https://cloud.google.com/alloydb/docs/ai/work-with-embeddings

**Comment by YanivHyper-Space** [comment]
Hyperspace 
[Hyperspace.json](https://github.com/superlinked/VectorHub/files/15185100/Hyperspace.json)
https://www.hyper-space.io/

**Reply by dhruv-anand-aintech** [reply]
Thanks for suggesting, and providing the values @YanivHyper-Space!

Could you send it in the form of a PR (similar to https://github.com/superlinked/VectorHub/pull/354), so that we can point out any changes needed

Thanks.

**Reply by YanivHyper-Space** [reply]
Thanks for the reply :)
Hyperspace DB is not currently in the list, meaning this is a new file to be added.

I've been trying to create pull request o add this file, but:
- I cannot add files (getting an error that only collaborators can).
- cannot add the new .json to the new pull request (same message).

Would appreciate any help

**Reply by dhruv-anand-aintech** [reply]
sure, you can fork the VectorHub repository, add your file, and then create a pull request to this repository.

https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project#forking-a-repository

**Reply by YanivHyper-Space** [reply]
Pull request created. Thanks for the help!

**Comment by Ratgor** [comment]
How about [SurrealDB](https://surrealdb.com/)?

**Comment by Ratgor** [comment]
How about [ArangoDB](https://arangodb.com/)

**Comment by Ratgor** [comment]
How about [ArcadeDB](https://arcadedb.com/)?
